source/SIL.AppBuilder.Portal.Frontend/src/ui/components/inputs/date-range/close-icon.tsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import CloseIcon from '@material-ui/icons/Close';
+
+export default class extends React.PureComponent {
```

Comment:
http://lucybain.com/blog/2018/react-js-pure-component/
I guess we should use more of this.

Reply:
+1